ngraph/test/runtime/ie/unit_test.manifest

```diff
@@ -1515,6 +1515,23 @@ gather_v7_axis_0_bool
 gather_v7_3d_indices_axis_1_batch_dims_1_int32
 gather_v7_4d_data_axis_2_batch_dims_1_int32
 
+# not yet implemented on CPU/GPU GatherND_8
+gather_nd_v8_single_indices
```

Comment:
Please use SLT instead

Reply:
Now we don't need to mark these tests as failed in plugins.